kod/util/rntmaint.kod

```diff
@@ -24,6 +27,29 @@ resources:
    RoomMaintenance_no_copy_copy = "You may not copy a copy of a room key."
    RoomMaintenance_copy_no_money = "You cannot afford a copy of your room key."
 
+   RoomMaintenance_command_room = "room"
+   RoomMaintenance_command_rent = "rent"
+   RoomMaintenance_command_cost = "cost"
+   RoomMaintenance_command_copy_key = "copy key"
+
+   RoomMaintenance_innkeeper_say_room_rented = \
+      "Here you go.  I hope you enjoy your stay, and if you have any problems "
+      "I will be glad to help you."
+   RoomMaintenance_innkeeper_say_cost = \
+      "I charge %i shillings to ~Irent~n a ~Iroom~n here for 85 days.  "
+      "If you want to stay longer, I charge %i per day if you pre-pay.  "
+      "Key copies cost %i shillings to make."
+   RoomMaintenance_innkeeper_say_days_left = \
+      "You like it here, I see.  There are %i days until you should "
+      "pay me again."
+   RoomMaintenance_innkeeper_say_too_long = \
+      "Oh, I can't be responsible for holding so much money!  Pay me again "
+      "later when you don't have so many days left on your tab."
+   RoomMaintenance_innkeeper_say_destroy_key_copy = \
+      "%q tells you, \"Oh, yes. I can get rid of this key for "
```

Comment:
Woops, good catch.  I was actually going to implement it as a template (seen in the latest update) so that each innkeeper's response didn't require all that extra formatting (~kMorrigan tells you, \"blah blah\") to make it all less error prone should somebody add rentable rooms to any other npcs later on.  Somehow, I forgot about that, but now it's set up and working.  All the other issues should be addressed, too.